aws-api-payload.json

```diff
@@ -156,7 +156,7 @@
                     "spark.yarn.dist.files": "/etc/spark/conf/hive-site.xml,/etc/hbase/conf/hbase-site.xml",
                     "spark.driver.extraJavaOptions": "-XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='{proxy_host}' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='instance-data.$REGION.compute.internal|*.s3.$REGION.amazonaws.com|s3.$REGION.amazonaws.com|sns.$REGION.amazonaws.com|sqs.$REGION.amazonaws.com|$REGION.queue.amazonaws.com|glue.$REGION.amazonaws.com|sts.$REGION.amazonaws.com|*.$REGION.compute.internal|dynamodb.$REGION.amazonaws.com' -Dhttps.proxyHost='{proxy_host}' -Dhttps.proxyPort='3128'",
                     "spark.driver.extraClassPath": "/usr/lib/hadoop-lzo/lib/*:/usr/lib/hadoop/hadoop-aws.jar:/usr/share/aws/aws-java-sdk/*:/usr/share/aws/emr/emrfs/conf:/usr/share/aws/emr/emrfs/lib/*:/usr/share/aws/emr/emrfs/auxlib/*:/usr/share/aws/emr/goodies/lib/emr-spark-goodies.jar:/usr/share/aws/emr/security/conf:/usr/share/aws/emr/security/lib/*:/usr/share/aws/hmclient/lib/aws-glue-datacatalog-spark-client.jar:/usr/share/java/Hive-JSON-Serde/hive-openx-serde.jar:/usr/share/aws/sagemaker-spark-sdk/lib/sagemaker-spark-sdk.jar:/usr/share/aws/emr/s3select/lib/emr-s3-select-spark-connector.jar:/usr/lib/hive/lib/hive-hbase-handler.jar:/usr/lib/hbase/*:/usr/lib/hive/lib/metrics-core-2.2.0.jar:/usr/lib/hive/lib/htrace-core-3.1.0-incubating.jar",
-                    "spark.sql.warehouse.dir": "s3://$ADG_S3_PUBLISH_BUCKET/data/external",
+                    "spark.sql.warehouse.dir": "s3://$S3_PUBLISH_BUCKET/data/external",
```

Comment:
Should be `s3://$S3_PUBLISH_BUCKET/analytical-dataset/data/external` for now; I reserve the right to change my mind once I see what all of these different outputs actually look like :)

Reply:
done